ci-operator/config/kubevirt/hyperconverged-cluster-operator/kubevirt-hyperconverged-cluster-operator-master.yaml

```diff
@@ -83,6 +83,19 @@ tests:
     RELEASE_DELTA=2 PREV=true make upgrade-test
   openshift_installer_src:
     cluster_profile: azure4
+- artifact_dir: /tmp/artifacts
+  as: hco-e2e-deploy-nightly-master-aws
+  commands: |
+    oc apply -f https://raw.githubusercontent.com/kubevirt/node-maintenance-operator/master/deploy/deployment-ocp/catalogsource.yaml
```

Comment:
Now we don't need this anymore due because we are postponing it as for https://bugzilla.redhat.com/show_bug.cgi?id=1874822

Reply:
Done